lib/backend/syncersv1/updateprocessors/networkpolicyprocessor.go

```diff
@@ -65,6 +65,7 @@ func convertPolicyV2ToV1Spec(spec apiv2.PolicySpec) (interface{}, error) {
 		Selector:      spec.Selector,
 		DoNotTrack:    spec.DoNotTrack,
 		PreDNAT:       spec.PreDNAT,
+		ApplyOnForward: spec.ApplyOnForward,
```

Comment:
indent looks off, may run `go fmt`?

Reply:
Strange, my editor shows correct indent. Will fix that.